node/pkg/vaa/structs.go

```diff
@@ -271,11 +271,15 @@ func Unmarshal(data []byte) (*VAA, error) {
 		return nil, fmt.Errorf("failed to read commitment: %w", err)
 	}
 
-	payload := make([]byte, 1000)
+	payload := make([]byte, 1001)
```

Comment:
@hendrikhofstadt can you provide some context on why this was set to 1000?

Reply:
Feedback I heard on this was that Solana has a limit of 1200 or so, so this seems like the rationale.